src/dcat-us/index.ts

```diff
@@ -0,0 +1,21 @@
+import { IItem } from '@esri/arcgis-rest-portal';
+import { FeedFormatterStream } from './feed-formatter-stream';
+
+export function getDataStreamDcatUs11(siteItem: IItem) {
+  console.log(siteItem);
```

Comment:
Just a friendly reminder to remove this when you can 😁

Reply:
Thank you! Yeah, just my attempt to get around Typescript's checks.